suite-native/settings/src/settingsSlice.ts

```diff
@@ -53,11 +54,20 @@ export const appSettingsSlice = createSlice({
 export const selectFiatCurrencyCode = (state: SettingsSliceRootState) =>
     state.appSettings.fiatCurrencyCode;
 export const selectBitcoinUnits = (state: SettingsSliceRootState) => state.appSettings.bitcoinUnits;
+export const selectAreSatsAmountUnit = (state: SettingsSliceRootState) =>
+    selectBitcoinUnits(state) === PROTO.AmountUnit.SATOSHI;
 export const selectIsOnboardingFinished = (state: SettingsSliceRootState) =>
     state.appSettings.isOnboardingFinished;
 export const selectViewOnlyCancelationTimestamp = (state: SettingsSliceRootState) =>
     state.appSettings.viewOnlyCancelationTimestamp;
 
+export const selectIsValueInSats = (
+    state: SettingsSliceRootState,
+    networkSymbol: NetworkSymbol | null | undefined,
```

Comment:
Why to allow null and undefined for networkSymbol?

Reply:
Just to make calling the selector easier. Most places where it's used have potentially `null` or `undefined` symbol. 😕 How would you suggest to change this?

Reply:
I see. I have not checked all usages before. In this case it makes sense 👍 , sorry for the comment! :)